docs/getting-started.md

```diff
@@ -8,14 +8,42 @@ make tutorial-image  # create the Docker image for the tutorial
 make tutorial-run
 ```
 
-Once the Ghidra server has started and the Jupyter notebook is up (this should take about one minute), follow the displayed instructions to access the Jupyter noteboks and have fun!
+It takes a minute for the notebook to start up. The jupyter notebook URL and token will finally print on the terminal.
+Once running, you can access the tutorial from [localhost:8888](http://localhost:8888) with your web browser. Have fun!
+
+## GUI
+OFRAK comes with a web-based GUI frontend for visualizing and manipulating binary targets. To activate the GUI from the
+OFRAK tutorial container, run:
+
+```
+docker exec -it ofrak-tutorial \
+  bash -c "nginx; python /ofrak_server.py 0.0.0.0 8877"
+```
+
+Once running, access [localhost:80](http://localhost:80) and start by dropping anything you'd like into it!
```

Comment:
Is it worth mentioning that port 80 has to already be mapped out in the `docker run` command?

Reply:
I see that you changed the `make run` rule, which is really helpful for this! I am not sure this comment still applies. I'm on the fence.

On one hand, you want to be clear that it's required. On the other, you don't want to over-explain for someone just using `make run`. I defer to you.